state.go

```diff
@@ -612,8 +612,9 @@ func (ue *UpdateErrorState) Handle(ctx *StateContext, c Controller) (State, bool
 // update is deemed as failed.
 type UpdateStatusReportState struct {
 	CancellableState
-	update UpdateResponse
-	status string
+	update             UpdateResponse
+	status             string
+	triesSendingReport int
```

Comment:
The reason is this line: https://github.com/mendersoftware/mender/pull/92/files#diff-a987904f2c16109d2be403c8d8d7f93eR703. If the waiting loop will be interrupted we won't reset the counter.